src/kirby/components/floating-action-button/floating-action-button.component.scss

```
@@ -3,10 +3,25 @@
 // Ionic fab can be customized by using the css properties listed in the CSS Custom Properties section of the below docs page:
 // https://ionicframework.com/docs/api/fab-button
 
+@mixin fab-size($fab-size) {
+    width: #{$fab-size}px;
+    height: #{$fab-size}px;
+}
+
 ion-fab-button {
     --box-shadow: 0 10px 20px -10px #{get-color('primary-shade')};
-    width: 64px;
-    height: 64px;    
+    
+    &.sm {
+        @include fab-size(size('xl'));
```

Consider readability...
```suggestion
        width: size('xl');
        height: size('xl');
```

---

I think it was readable enough (perhaps we could rename the mixin), but i'll change it to your suggestion